lib/reporter.js

```diff
@@ -250,9 +250,11 @@ class AllureReporter extends events.EventEmitter {
                 const currentSuite = allure.getCurrentSuite()
                 const test = allure.getCurrentTest()
 
-                const browserName = suite.runner[suite.cid].browserName || suite.cid
-                const version = suite.runner[suite.cid].version || ''
-                test.addParameter('argument', 'browser', `${browserName}-${version}`)
+                const { browserName, deviceName } = suite.runner[suite.cid] // || suite.runner[suite.cid].deviceName || suite.cid
```

Comment:
1) Also needs change in 
https://github.com/webdriverio-boneyard/wdio-allure-reporter/blob/c4deeade4de600d290987623d429f4925d5b778c/lib/reporter.js#L87-L90
2) can you extract this to separeate method and use in both places?
3) Needs tests

Reply:
Hello Boris, thanks for the review. Indeed I missed that one (and forgot a dumb comment).

I just pushed changes, please let me know if that's ok for you.
I will add tests probably around monday as I will be abroad on thursday and friday.

Reply:
Hi Kevin, changes looks good.